kedro-datasets/kedro_datasets/spark/__init__.py

```diff
@@ -10,6 +10,7 @@
 SparkHiveDataset: Any
 SparkJDBCDataset: Any
 SparkStreamingDataset: Any
+GBQQueryDataset: Any
```

Comment:
Nit: Can you move this above `SparkDataset` to maintain the existing alphabetical order?